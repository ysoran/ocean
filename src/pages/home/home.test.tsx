import React from 'react';
import { shallow } from 'enzyme';
import Home from './home';

const setup = () => {
    const wrapper = shallow(<Home/>);
    return {
        wrapper,
    };
};

describe('Home component', () => {
    it('should render snapshot', () => {
        const { wrapper } = setup();
        expect(wrapper).toMatchSnapshot();
    });
});
