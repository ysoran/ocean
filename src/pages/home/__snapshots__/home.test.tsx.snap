// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render snapshot 1`] = `ShallowWrapper {}`;
